presto-hive-common/src/main/java/com/facebook/presto/hive/aws/security/AWSSecurityMappings.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.aws.security;
+
+import com.facebook.presto.hive.aws.security.lakeformation.AWSLakeFormationSecurityMapping;
+import com.facebook.presto.hive.aws.security.s3.AWSS3SecurityMapping;
+import com.facebook.presto.spi.security.ConnectorIdentity;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public class AWSSecurityMappings
+{
+    private final List<AWSLakeFormationSecurityMapping> awsLakeFormationSecurityMappings;
+    private final List<AWSS3SecurityMapping> awsS3SecurityMappings;
+
+    @JsonCreator
+    public AWSSecurityMappings(
+            @JsonProperty("lakeformation") List<AWSLakeFormationSecurityMapping> awsLakeFormationSecurityMappings,
+            @JsonProperty("s3") List<AWSS3SecurityMapping> awsS3SecurityMappings)
+    {
+        checkArgument(
```

Comment:
In my initial design, I had both of them(AWS S3 and AWS Lake Formation) as separate features, which meant both had their own set of configs to enable the feature. I felt it would be similar code for both the features and since they are a subset of AWS services, we could have a common feature that can be used either for S3 or Lake Formation depending upon the use-case.

Since only one constructor can be annotated with `@JsonCreator`, we had to do it this way since we use this class to parse the Json File which has the mappings.